app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
 {
+  "accept": {
+    "message": "Aceptar"
+  },
+  "account": {
+    "message": "Cuenta"
+  },
+  "accountDetails": {
+    "message": "Detalles de la cuenta"
+  },
+  "accountName": {
+    "message": "Nombre de la cuenta"
+  },
+  "address": {
+    "message": "Direccion"
+  },
+  "addToken": {
+    "message": "Agregar Token"
+  },
+  "amount": {
+    "message": "Cantidad"
+  },
+  "amountPlusGas": {
+    "message": "Cantidad + Gas"
+  },
+  "appDescription": {
+    "message": "Extension del explorador usar Ethereum",
+    "description": "La descripcion de la aplicacion"
+  },
   "appName": {
     "message": "MetaMask",
-    "description": "The name of the application"
+    "description": "El nombre de la aplicacion"
   },
-  "appDescription": {
-    "message": "Administración de identidad en Ethereum",
-    "description": "The description of the application"
+  "attemptingConnect": {
+    "message": "Intentando conectar a la Blockchain"
```

Comment:
I agree, we don't have neuters genders in spanish: Everything is either a _he_ or a _she_. As we say `la cadena`, the (she) chain, by transitivity it is right to say, `la cadena de bloques`, the (she) blockchain.

I am adding this note to create awareness and precedent :-)

Reply:
Nice